### PR TITLE
Create .asf.yaml

### PR DESCRIPTION
**Pull request by qqeasonchen**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - event-driven
+    - event-streaming
+    - event-sourcing
+    - event-governance
+    - cloud-native
+    - serverless
+    - workflow-serverless
+    - esb
+    - message-bus
+    - CQRS
```
Comment (kezhenxu94) [inline .asf.yaml:33]: I remember capital characters are not allowed in the labels.

File: .asf.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  enabled_merge_buttons:
+    squash:  true
+    merge:   false
+    rebase:  false
+  protected_branches:
+    master:
```
Comment (kezhenxu94) [inline .asf.yaml:39]: Do you want to switch the protected branch from develop to master?

**Review by keranbingaa — APPROVED** [review]

File: .asf.yaml (hunk 1 above)
Comment (qqeasonchen) [reply .asf.yaml:33]: I think it is ok.

File: .asf.yaml (hunk 2 above)
Comment (qqeasonchen) [reply .asf.yaml:39]: It is.

**Review by xwm1992 — APPROVED** [review]

File: .asf.yaml (hunk 1 above)
Comment (kezhenxu94) [reply .asf.yaml:33]: > I think it is ok.

By “not allowed”, I mean it causes the settings failed to take effect

File: .asf.yaml (hunk 2 above)
Comment (qqeasonchen) [reply .asf.yaml:39]: > Do you want to switch the protected branch from develop to master?

If i cancel the protection from all branches, committers can modify the code directly or only PMC? i did not get any instructions from official docs. thanks.

Comment (kezhenxu94) [reply .asf.yaml:39]: > > Do you want to switch the protected branch from develop to master?
> 
> If i cancel the protection from all branches, committers can modify the code directly or only PMC? i did not get any instructions from official docs. thanks.

Committers and (P)PMC members. People with write access to the repo can directly push to it.

Comment (qqeasonchen) [reply .asf.yaml:39]: > > > Do you want to switch the protected branch from develop to master?
> > 
> > 
> > If i cancel the protection from all branches, committers can modify the code directly or only PMC? i did not get any instructions from official docs. thanks.
> 
> Committers and (P)PMC members. People with write access to the repo can directly push to it.

Got，thanks.

**Pull request merged**